public/kibi_timeline.js

```diff
@@ -149,25 +149,29 @@ define(function (require) {
         const groupId = group.id;
         const groupColor = group.color;
 
+        searchSource.highlight({
+          fields: {
+            '*': {
```

Comment:
The highlighted terms show how/why a document matched the saved search. Maybe the user wants to display another field in the label and still see why the document matched the search